vision_agent/tools/tools.py

```diff
@@ -652,6 +653,50 @@ def florencev2_object_detection(image: np.ndarray, prompt: str) -> List[Dict[str
     return return_data
 
 
+def florencev2_ocr(image: np.ndarray) -> List[Dict[str, Any]]:
+    """'florencev2_ocr' is a tool that can detect text in an image without any text
```

Comment:
"""'florencev2_ocr' is a tool that can detect text and text regions in an image. Each text region contains one 
    line of text. It returns a list of detected text, text region as bounding boxes with normalized coordinates,
    and confidence scores.